src/Core/src/Platform/iOS/MauiScrollView.cs

```diff
@@ -6,10 +6,25 @@ namespace Microsoft.Maui.Platform
 {
 	public class MauiScrollView : UIScrollView
 	{
+		readonly WeakEventManager _weakEventManager = new WeakEventManager();
+
 		public MauiScrollView()
 		{
 		}
 
+		public event EventHandler LayoutSubviewsChanged
```

Comment:
I don't think we want to live with this public API just yet. 
Also I wonder if is better to use the proxy approach . any opinions @jonathanpeppers  ? 

I think we could also pass the handler  to this view add a WeakReference to it , and use to call 	`scrollViewHandler.Invoke(nameof(IScrollView.RequestScrollTo), scrollViewHandler.PendingScrollToRequest);`

Reply:
Does this test pass if you remove `WeakEventManager`?

https://github.com/dotnet/maui/blob/cd64c827bf98969cf980b642bc60688e3a9506d3/src/Controls/tests/DeviceTests/Elements/ScrollView/ScrollViewTests.cs#L81-L82

If it does, we can make it a regular event.

In either case, we should add this for now:
```suggestion
		[UnconditionalSuppressMessage("Memory", "MA0001", Justification = "Proven safe in test: ScrollViewTests.DoesNotleak")]
		public event EventHandler LayoutSubviewsChanged
```

Reply:
> Does this test pass if you remove `WeakEventManager`?
> 
> https://github.com/dotnet/maui/blob/cd64c827bf98969cf980b642bc60688e3a9506d3/src/Controls/tests/DeviceTests/Elements/ScrollView/ScrollViewTests.cs#L81-L82
> 
> If it does, we can make it a regular event.
> 
> In either case, we should add this for now:

Is passing without the `WeakEventManager`. Changed and added the suppress message.

Reply:
<img width="362" alt="Captura de pantalla 2023-07-28 a las 12 52 44" src="https://github.com/dotnet/maui/assets/6755973/dbc41a4c-ee18-44ae-ad9c-aad73f86d6c4">